hasher-matcher-actioner/hmalib/lambdas/fetcher.py

```diff
@@ -48,14 +49,24 @@ class FetcherConfig:
     s3_bucket: str
     s3_te_data_folder: str
     collab_config_table: str
+    data_store_table: str
 
     @classmethod
     @lru_cache(maxsize=1)  # probably overkill, but at least it's consistent
     def get(cls):
+        # These defaults are naive but can be updated for testing purposes.
         return cls(
-            s3_bucket=os.environ["THREAT_EXCHANGE_DATA_BUCKET_NAME"],
-            s3_te_data_folder=os.environ["THREAT_EXCHANGE_DATA_FOLDER"],
-            collab_config_table=os.environ["THREAT_EXCHANGE_CONFIG_DYNAMODB"],
+            s3_bucket=os.environ.get(
+                "THREAT_EXCHANGE_DATA_BUCKET_NAME",
+                "DEFAULT_BUCKET",
```

Comment:
Is this a magic value?

Shouldn't we stall execution if this value is not provided?

Reply:
the fetcher can be used locally if the values are overridden. I'd prefer having a value here for ease of debugging and quickly changing/testing. Are you alright with `None` or want to me to get rid of the `.get` calls all together?

Reply:
None is fine. None is magic, but it is sensible magic.